src/components/PupilComponents/LessonEngineProvider/LessonEngineProvider.tsx

```diff
@@ -210,11 +223,16 @@ export const LessonEngineProvider = memo(
       currentSection: initialSection,
       lessonStarted: false,
       sections: {},
+      timeStamp: {
+        section: initialSection,
+        time: new Date().getTime(),
+      },
     });
     const navigateToSection = useNavigateToSection();
 
     // Sync the currentSection when the user navigates back/forward
     useLessonPopStateHandler((section) => {
+      console.log("useLessonPopStateHandler", section);
```

Comment:
can we strip out any console.logs please

Reply:
oops!